tests/src/assembler_xtheadcondmov_tests.cpp

```diff
@@ -0,0 +1,33 @@
+#include <catch/catch.hpp>
```

Comment:
perhaps rename this file to `assembler_xthead_tests.cpp` so it makes adding further xthead tests a little more straightforward without needing to add more cpp files (mainly because I'll add support for the rest of the Xthead instructions in the assembler, so it saves me a little busywork).